examples/step-45/doc/intro.dox

```diff
@@ -1,72 +1,142 @@
 <a name="Intro"></a>
 <h1>Introduction</h1>
 
-In this example we consider how to use periodic boundary conditions in
-deal.II. Periodic boundary conditions are a typical approach when one wants to
-solve some equation on a representative piece of a larger domain that repeats
-in one or more direction; an example is the simulation of the electronic
-structure of photonic
+In this example we present how to use periodic boundary conditions in
+deal.II. Periodic boundary conditions are algebraic constraints that
+typically occur in computations on representative regions of a larger
+domain that repeat in one or more directions.
+
+An example is the simulation of the electronic structure of photonic
 crystals, because they have a lattice-like structure and, thus, it often
-suffices to do the actual computation on only one cell of the lattice. To be
-able to proceed this way one has to assume that the computation can be periodically
-extended to the other cells. This requires the solution to be periodic with respect
-to the cells. Hence the solution has to obtain the same nodal values on opposite
-parts of the boundary. In the figure below we show this
-concept in two space-dimensions. There, all dashed faces with the same color should
-have the same boundary values:
+suffices to do the actual computation on only one cell of the lattice. To
+be able to proceed this way one has to assume that the computation can be
+periodically extended to the other cells; this requires the solution to
+have a periodic structure.
 
-<img src="http://www.dealii.org/images/steps/developer/step-45.periodic_cells.png" alt="">
+<a name="Procedure"></a>
+<h1>Procedure</h1>
 
-To keep things simple, in this tutorial we will consider an academic,
-simplified problem that allows us to focus on only that part that we are
-interested in here, namely how to set up periodic boundary
-conditions. Specifically, we solve the Poisson problem on a domain, where the
-left and right parts of the boundary are identified. Let $\Omega=(0,1)^2$ and
-consider the problem
-@f{align*}
-   -\Delta u &=
-   \cos(2\pi x)e^{-2x}\cos(2\pi y)e^{-2y}  \qquad &&\text{in }\Omega
-  \\ 
-   u(x,0) &= 0 \qquad &&\text{for }x\in(0,1)\qquad &&\text{(bottom boundary)}
-  \\
-   u(x,1) &= 0 \qquad &&\text{for }x\in(0,1)\qquad &&\text{(top boundary)}
-  \\
-   u(0,y) &= u(1,y) \qquad &&\text{for }y\in(0,1) 
-   \qquad && \text{(left and right boundaries)}
-@f}
-Note that the source term is not symmetric and so the solution would not be
-periodic unless this is explicitly enforced.
+deal.II provides a number of high level functionality to impose periodic
+boundary conditions.
+The general approach to apply periodic boundary conditions consists of
+three steps (see also the
+@ref GlossPeriodicConstraints "Glossary entry on periodic boundary conditions"):
+-# Create a mesh
+-# Gather the periodic faces using GridTools::collect_periodic_faces()
```

Comment:
Instead of the unspecific "gather" maybe use "identify those pairs of faces on different parts of the boundary across which the solution should be symmetric, using GridTools::..."?